core/exception/interrupt_spec.rb

```diff
@@ -0,0 +1,35 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe "Interrupt.new" do
```

Comment:
Yes, I think it is OK to duplicate in this case,
because it also explains the next specs and why Interrupt responds to `signo`, `signm` etc.